test/e2e/scenario/rum/actions.scenario.ts

```diff
@@ -79,9 +81,51 @@ describe('action collection', () => {
           name: 'click me',
         },
         type: 'click',
+        frustration_type: [],
       })
 
       expect(resourceEvents.length).toBe(1)
       expect(resourceEvents[0].action!.id).toBe(actionEvents[0].action.id!)
     })
+
+  createTest('collect an "error click"')
+    .withRum({ trackInteractions: true, enableExperimentalFeatures: ['frustration-signals'] })
+    .withBody(
+      html`
+        <button>click me</button>
+        <script>
+          const button = document.querySelector('button')
+          button.addEventListener('click', () => {
+            button.setAttribute('data-clicked', 'true')
+            throw new Error('Foo')
+          })
+        </script>
+      `
+    )
+    .run(async ({ serverEvents }) => {
+      const button = await $('button')
+      await button.click()
+      await flushEvents()
+      const actionEvents = serverEvents.rumActions
+
+      expect(actionEvents.length).toBe(1)
+      expect(actionEvents[0].action.frustration_type).toEqual(['error'])
+      expect(actionEvents[0].action.error!.count).toBe(1)
+      await withBrowserLogs((browserLogs) => {
+        expect(browserLogs.length).toEqual(1)
+      })
+    })
+
+  createTest('collect a "dead click"')
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Could it be interesting to have a test for multiple frustrations on the same click?